packages/chips/Chip.tsx

```diff
@@ -26,19 +26,21 @@ import * as Ripple from '@material/react-ripple';
 import {MDCChipFoundation} from '@material/chips/dist/mdc.chips';
 
 export interface ChipProps extends Ripple.InjectedProps<HTMLDivElement> {
-  id?: string;
+  id: string;
```

Comment:
This change is pending #649

Reply:
and had #649 been approved... ;)

I'll change this back.